ecommerce/courses/models.py

```diff
@@ -129,6 +129,10 @@ def enrollment_code_product(self):
 
     def get_course_seat_name(self, certificate_type, id_verification_required):
         """ Returns the name for a course seat. """
+        if getattr(settings, 'ENABLE_NEW_COURSE_SEAT_NAME', False):
+            name = u'Seat in the Pearson Advance {} course'.format(self.name)
```

Comment:
@Jacatove, what do you think, does this sound better?
```suggestion
            name = u'Seat in Pearson Advance {} course'.format(self.name)
```

Reply:
@ivanvgh I thought the same as you, but on the ticket is Seat in the Pearson...